Example/src/LightboxExample.js

```diff
@@ -138,7 +119,7 @@ function ImageTransition({ activeImage, onClose }) {
 
   const targetX = useSharedValue(0);
   const targetY = useSharedValue(
-    (dimensions.height - targetHeight) / 2 - HEADER_HEIGHT
+    (dimensions.height - targetHeight) / 2 - useHeaderHeight()
```

Comment:
Could you move it above and assign it to variable? I think hooks should be near the top of a component as a convention.